### ISSUE TITLE
async module does not work in the browser

### ISSUE THREAD

**Issue by iest** [ForbesLindesay/work-token#1]
Using `var work = require('work-token/async');` fails, as it pulls in `work-token/async` instead of `work-token/lib/async-browser`, which I guess is the desired behaviour?

Specifically, `child_process` is not defined in the browser.

When I try to pull in `work-token/lib/async-browser` it also fails, because `fs` is not defined in the browser either.

Am I doing something wrong with browserify?

`sync` version works fine — but obviously blocks on the client until the token's generated :)

**Comment by ForbesLindesay** [comment]
I'm guessing you were installing from the GitHub repo, which won't work.  Unfortunately I had broken the npm version so it didn't actually include any of the files, and also didn't always resolve the browser version of async correctly.  I've fixed these bugs and published 1.0.2.  The redirections now look like:

``` js
  "browser": {
    "./async": "./bin/async-browser.js",
    "./async.js": "./bin/async-browser.js",
    "./lib/sha512.js": "./lib/sha512-browser.js"
  },
```

When you `require('work-token/async')` in your browser it will load `work-token/bin/async-browser.js`, which is compiled from `./lib/async-browser.js`.

The tricky part is that it inlines the source code for the worker to make the module easier to require and use.

**Comment by ForbesLindesay** [comment]
I've done a pull request to make the changes to i-am-the-fold https://github.com/iest/i-am-the-fold/pull/6
